app/src/main/java/com/sdp/movemeet/models/Activity.java

```diff
@@ -84,7 +84,7 @@ public Activity(String activityId,
         this.longitude = longitude;
         this.latitude = latitude;
         this.description = description;
-        super.documentPath = documentPath;
+        this.documentPath = documentPath;
```

Comment:
Yes, I also updated this on my side! This allows us in fact to retrieve the documentPath of the activities without any problems! Using your `FirebaseObject` class as an abstract class rather than an interface would have been probably slightly cleaner, but in the end, I think going back to an interface is reasonable 👍.